include/Spectra/LinAlg/Orthogonalization.h

```diff
@@ -42,6 +54,10 @@ void QR_orthogonalisation(Matrix& in_output)
     in_output = qr.householderQ() * I;
```

Comment:
Conceptually, I feel
```cpp
in_output.leftCols(ncols).noalias() = qr.householderQ() * I;
```
is more appropriate, since we don't want to discard columns of `in_output` if `ncols > nrows`.

But this is a minor issue since in the actual code `ncols` is never larger than `nrows`.